.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--
+See http://loopback.io/doc/en/contrib/triaging-pull-requests.html to understand
+the MINIMUM expections for pull requests.
+-->
+
+### Bug or feature
+
+- [ ] Bug
+- [ ] Feature
```

Comment:
Why is this important? Shouldn't this be obvious from the pull request description?

IMO, backwards compatibility is the most important concern to address when sending a pull request.

```
 - [ ] backwards compatible
 - [ ] breaking change(s)
```